fvm/contractFunctionInvocations.go

```diff
@@ -150,6 +149,39 @@ func AccountStorageCapacityInvocation(
 	}
 }
 
+// AccountsStorageCapacityInvocation prepares a function that calls get storage capacity on the storage fees contract
+// for multiple accounts at once
+func AccountsStorageCapacityInvocation(
+	env Environment,
+	traceSpan opentracing.Span,
+) func(addresses []common.Address) (cadence.Value, error) {
+	return func(addresses []common.Address) (cadence.Value, error) {
+		arrayValues := make([]cadence.Value, len(addresses))
```

Comment:
can we check nil or empty of `addresses` here to save one invocation if no valid address passed in?

Reply:
I am not sure if I want to put this logic here. This is just a wrapper to call the smart contract function. Any conditions like that should be either in the smart contract or better yet (performance wise) in the function calling this one. 

I will check that the function calling this one doesn't do it with an empty array.

What do you think?